data/DataTrans.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import pandas as pd
+import numpy as np
+import csv
+Max_Value=3000
+Min_Value=0
+
+def get_Zone(Z_csv):
+    zone=pd.read_csv(Z_csv)
+
+    # Create dictionary to store the mapping of states and codes
+    zone_dic={}
+    for i in range(len(zone)):
+        zone_dic[zone['STATE'][i]]=zone['ID'][i]
+    return zone_dic
+
+
+# Clean data; throw substations which are outside the United States and not available.
+def Clean(E_csv,zone_dic):
+    csv_data = pd.read_csv(E_csv)
+    Num_sub=len(csv_data)
+    row_indexs=[]
+    for i in range(Num_sub):
+        if((csv_data['STATE'][i] not in zone_dic) or (csv_data['STATUS'][i] != 'IN SERVICE') ):
+            row_indexs.append(i)
+    clean_data=csv_data.drop(labels=row_indexs)
+    return clean_data
+
+
+
+#Create dictionary to store neighbors of each sub
+def Neighbors(T_csv):
+    Trans_data = pd.read_csv(T_csv, usecols=['VOLTAGE','SUB_1','SUB_2'])
+    N_dict={}
```

Comment:
We use `=` without space and with space - can we unify to always use space before and after the `=`? i.e. use `N_dict = {}` instead of `N_dict={}`

Reply:
In a couple of our other repos, we have used `black` to standardize formatting, and set it up to auto-run for each PR.

Reply:
Short answer: we do `N_dict = {}`, and for parameters passing into a function, there are no spaces: `Function(a=123)`. And these conventions are defined by `black` automatically as well.

Reply:
Sorry didn;t notice that. Thanks for your advice, I will correct it.